LICENSE

```diff
@@ -198,4 +198,4 @@
    distributed under the License is distributed on an "AS IS" BASIS,
```

Comment:
Changes in unreleased.md file are missing. Could you please add changelog there?

Reply:
Updated